io/driver/module_unload_load.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+CONFIG_FILE="$AVOCADO_TEST_DATADIR"/config
+DRIVER=`lspci -k | grep -iw "Kernel driver in use" | cut -d ':' -f2 | sort | uniq`
+
+#Loads the kernel driver modules
+module_load() {
+    echo "Reloading driver $1"
+    modprobe $1
+    if [ $? != 0 ]; then
+        echo "Failed to load driver module $1"
+        break;
+    fi
+    echo
+}
+
+#Unloads the kernel driver modules
+module_unload() {
+    for i in $(cat $CONFIG_FILE | grep "$1=" | awk -F'=' '{print $2}'); do
+        module_unload $i
+        if [ $? != 0 ]; then
+            echo "Failed to unload driver module $i"
+            return
+        fi
+    done
+    echo "Unloaded driver $1"
+    rmmod $1
```

Comment:
Yep, I understand the point, but in reality the modules might not be unloaded as the `rmmod` could fail and the test won't complain about it so I'd prefer checking the error and failing the test in such case. But if you consider this test only as stress test, then fine, but it should be mentioned in the heading.

About the system, I simply run Fedora 23 on Fedora 23, x86_64, no special handling, but the failure is not important, the test behaved correctly. We might later expand some blacklist to avoid such problems, but it should be based on real-world usage, not just me trying the test.